src/test/scala/PSO/ParticleSpec.scala

```diff
@@ -0,0 +1,95 @@
+package PSO
+
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.WordSpecLike
+import org.scalatest.Matchers
+
+import com.trifectalabs.myriad.pso._
+import com.typesafe.config.ConfigFactory
+
+import akka.actor.ActorSystem
+import akka.actor.Props
+import akka.testkit.{ DefaultTimeout, ImplicitSender, TestKit }
+import scala.concurrent.duration._
+
+class ParticleSpec
+  extends TestKit(ActorSystem("ParticleSpec",
+    ConfigFactory.parseString(ParticleSpec.config)))
+  with DefaultTimeout with ImplicitSender
+  with WordSpecLike with Matchers with BeforeAndAfterAll {
+  //setup
+  def obj(sol: List[Double]): Double = {
+    sol.map(x => x*x).sum
+  }
+  val solutions = List(
```

Comment:
We might want to move this to some function.

`val solutions = initSolutions`

`def initSolutions = List(...)`